model_zoo/dac_ctr/feature_config.py

```diff
@@ -0,0 +1,141 @@
+STANDARDIZED_FEATURES = [
```

Comment:
We have a model `cenus_wide_deep` in the model zoo already. So how about naming the directory from `dac_ctr` to `criteo_wide_deep`? We may add `criteo_xdeepfm` in the later.

Reply:
The `census_wide_deep` is a demo to design feature transformation. We are considering to develop a tool for users to transform feature in #1950. We can place the `wide_deep` and `xdeepfm` model to a folder and use them in the train definition file for census and dac. We can first merge the PR and then refactor them.